src/Sylius/Behat/Page/Admin/ProductVariant/CreatePage.php

```diff
@@ -29,7 +30,15 @@ class CreatePage extends BaseCreatePage implements CreatePageInterface
      */
     public function specifyPrice($price, $channel)
     {
-        $this->getDocument()->fillField($channel, $price);
+        $this->getDocument()->fillField('Price for ' . $channel, $price);
```

Comment:
`specifyPrice($price, ChannelInterface $channel)`. Instead of concatenation try to use `sprintf('Price for %s', $channel->getName())`.